### PR TITLE
Internal: Make sure that work that landed in OS 12 also use update hook numbers in this range

### PR DESCRIPTION
**Pull request by robertragas**
## Problem
During the release of Open Social 12, there was a PR that was originally written for 11 but changed to 12. 
Update hook were therefore named as 11. To avoid confusion we need to update this to 12 so other developers don't continue on this number.

It should also be no issue that these hooks run again if they already did.

## Solution
Update the hook numbers from 11 to 12.

## Issue tracker
None - Internal

## Theme issue tracker
*[Required if applicable] Paste a link to the drupal.org theme issue queue item, either from [socialbase](https://www.drupal.org/project/socialbase) or [socialblue](https://www.drupal.org/project/socialblue). If any other issue trackers were used, include links to those too.*

## How to test
- [ ] Run drush updb and see that update hooks are still executed properly.

### Before merge
- [ ] Code/peer review is completed
- [ ] All commit messages are [clear and clean](https://open-social.slite.com/app/docs/DnmermZDIx_0OQ). If applicable a rebase was performed
- [ ] All automated tests are green
- [ ] Functional/manual tests of the acceptance criteria are approved
- [ ] All acceptance criteria were met
- [ ] New features or changes to existing features are covered by tests, either unit (preferably) or behat
- [ ] Update path is tested. New hook_updates should respect update order, right naming convention and consider hook_post_update code
- [ ] Module can be safely uninstalled. Update/implement hook_uninstall and make sure that removed configuration or dependencies are removed/uninstalled
- [ ] This pull request has all required labels (team/type/priority)
- [ ] This pull request has a milestone
- [ ] This pull request has an assignee (if applicable)
- [ ] Any front end changes are tested on all major browsers
- [ ] New UI elements, or changes on UI elements are approved by the design team
- [ ] New features, or feature changes are approved by the product owner

### After merge
- [ ] Code is tested on all branches that it has been cherry-picked
- [ ] Update hook number might need adjustment, make sure they have the correct order
- [ ] The Drupal.org ticket(s) are updated according to this pull request status

## Release notes
Internal: Re-organize the update hook numbers to make sure that work that landed in Open Social 12 also shows this in the code.

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3547-ujvr9tjfdzhnqwejwt8r11qfqabqoiwi.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/651d480ef57766838b4f2513

File: modules/social_features/social_group/modules/social_group_flexible_group/social_group_flexible_group.install [hunk 1]
```diff
@@ -640,13 +640,22 @@ function social_group_flexible_group_update_11901(): string {
   return $updater->logger()->output();
 }
 
+/**
+ * Disable revisions for flexible group.
+ */
+function social_group_flexible_group_update_111102(): void {
```
Comment (ronaldtebrake) [inline modules/social_features/social_group/modules/social_group_flexible_group/social_group_flexible_group.install:646]: should this also get a new number?

Comment (robertragas) [reply modules/social_features/social_group/modules/social_group_flexible_group/social_group_flexible_group.install:646]: This one has a milestone of 11.10.1
https://github.com/goalgorilla/open_social/pull/3517

Thats why I kept it as this same number.

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3547-ujvr9tjfdzhnqwejwt8r11qfqabqoiwi.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/651d480ef57766838b4f2513

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3547-ujvr9tjfdzhnqwejwt8r11qfqabqoiwi.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/651d480ef57766838b4f2513

File: modules/custom/social_gdpr/social_gdpr.install [hunk 2]
```diff
@@ -52,7 +52,7 @@ function social_gdpr_update_8902() {
  * Permissions to check:
  * - "translate data_policy".
  */
-function social_gdpr_update_111101(): void {
```
Comment (zanvidmar) [inline modules/custom/social_gdpr/social_gdpr.install:55]: This change should land on 11.x as it it BC. Hook update number should not be changed.

File: modules/social_features/social_book/social_book.install [hunk 3]
```diff
@@ -294,7 +294,7 @@ function social_book_update_11401(): string {
  * Permissions to check:
  * - "translate book node".
  */
-function social_book_update_111101(): void {
```
Comment (zanvidmar) [inline modules/social_features/social_book/social_book.install:297]: This change should land on 11.x as it it BC. Hook update number should not be changed.

File: modules/social_features/social_core/social_core.install [hunk 4]
```diff
@@ -1693,7 +1693,7 @@ function social_core_update_11900() : string {
  * Permissions to check:
  * - "access contextual links".
  */
-function social_core_update_111101(): void {
```
Comment (zanvidmar) [inline modules/social_features/social_core/social_core.install:1696]: This change should land on 11.x as it it BC. Hook update number should not be changed.

File: modules/social_features/social_group/modules/social_group_flexible_group/social_group_flexible_group.install [hunk 5]
```diff
@@ -640,13 +640,22 @@ function social_group_flexible_group_update_11901(): string {
...
+  $config->save();
+}
+
 /**
  * Search for invalid permission(s) and remove them from existing roles.
  *
  * Permissions to check:
  * - "translate flexible_group group".
  */
-function social_group_flexible_group_update_111101(): void {
```
Comment (zanvidmar) [inline modules/social_features/social_group/modules/social_group_flexible_group/social_group_flexible_group.install:649]: This change should land on 11.x as it it BC. Hook update number should not be changed.

File: modules/social_features/social_landing_page/social_landing_page.install [hunk 6]
```diff
@@ -810,7 +810,7 @@ function social_landing_page_update_111101(): void {
  * Permissions to check:
  * - "translate landing_page node".
  */
-function social_landing_page_update_111102(): void {
+function social_landing_page_update_120002(): void {
```
Comment (zanvidmar) [inline modules/social_features/social_landing_page/social_landing_page.install:813]: This change should land on 11.x as it it BC. Hook update number should not be changed.

File: modules/social_features/social_page/social_page.install [hunk 7]
```diff
@@ -263,7 +263,7 @@ function social_page_update_111101(): void {
  * Permissions to check:
  * - "translate page node".
  */
-function social_page_update_111102(): void {
```
Comment (zanvidmar) [inline modules/social_features/social_page/social_page.install:266]: This change should land on 11.x as it it BC. Hook update number should not be changed.

File: modules/social_features/social_page/social_page.install [hunk 8]
```diff
@@ -238,7 +238,7 @@ function social_page_update_10302() {
 /**
  * Remove deprecated permission "override page revision log entry".
  */
-function social_page_update_111101(): void {
```
Comment (zanvidmar) [inline modules/social_features/social_page/social_page.install:241]: This change should land on 11.x as it it BC. Hook update number should not be changed.

File: modules/social_features/social_landing_page/social_landing_page.install [hunk 9]
```diff
@@ -780,7 +780,7 @@ function social_landing_page_update_11401(): string {
  * - "override landing_page revision log entry",
  * - "override landing_page promote to front landing_page option".
  */
-function social_landing_page_update_111101(): void {
```
Comment (zanvidmar) [inline modules/social_features/social_landing_page/social_landing_page.install:783]: This change should land on 11.x as it it BC. Hook update number should not be changed.

File: modules/social_features/social_profile/social_profile.install [hunk 10]
```diff
@@ -472,7 +472,7 @@ function social_profile_update_11701(): string {
  * - "view profile",
  * - "add any profile profile".
  */
-function social_profile_update_111101(): void {
```
Comment (zanvidmar) [inline modules/social_features/social_profile/social_profile.install:475]: This change should land on 11.x as it it BC. Hook update number should not be changed.

File: modules/social_features/social_topic/social_topic.install [hunk 11]
```diff
@@ -501,7 +501,7 @@ function social_topic_update_11401(): string {
 /**
  * Remove deprecated permission "override topic revision log entry".
  */
-function social_topic_update_111101(): void {
```
Comment (zanvidmar) [inline modules/social_features/social_topic/social_topic.install:504]: This change should land on 11.x as it it BC. Hook update number should not be changed.

File: modules/social_features/social_topic/social_topic.install [hunk 12]
```diff
@@ -526,7 +526,7 @@ function social_topic_update_111101(): void {
  * Permissions to check:
  * - "translate topic node".
  */
-function social_topic_update_111102(): void {
```
Comment (zanvidmar) [inline modules/social_features/social_topic/social_topic.install:529]: This change should land on 11.x as it it BC. Hook update number should not be changed.

**Review by zanvidmar — CHANGES_REQUESTED** [review]
Some of this changes should land on 11.x as they are BC.
Hook update numbers should not be changed (I mark each one of them in comments)

**Comment by robertragas** [comment]
Closing the PR as it was decided to create 11.11.x

**Pull request closed without merging**